README.md

```diff
@@ -30,11 +34,14 @@ To get a local copy up and running follow these simple example steps.
 - 
 - In the cloned project folder, run 'npm run start'
 ### Usage
-- coming soon
+- click on a number button, then on an operation to get the result displayed on the grey bar.
+- click on A/C to clear all operations.
 ### Run tests
 - No tests yet
 ## I learned:
 - how to setup a react project using create-react-app
+- how to use states and props to pass information between components
+- how to handle onClick events
 ## Authors
```

Comment:
### **OPTIONAL**

_For a completely professional README file and repo, please consider this:_
- [ ] Kindly fix the MIT and issues page broken links
_This should have been brought up earlier, I'm sorry about that!_ 😄 🙏